pkg/rbac/clusterrole.go

```diff
@@ -44,7 +44,6 @@ func NewClusterRoleBuilder(apiClient *clients.Settings, name string, rule v1.Pol
 			ObjectMeta: metav1.ObjectMeta{
 				Name: name,
 			},
-			Rules: []v1.PolicyRule{rule},
```

Comment:
Why does it need to be removed?

Reply:
I agree this should be removed.  When I was trying to just run a simple `Delete()` in my project I had to populate a rule prior to issuing a delete.

Reply:
@trewest It's  bug that i've found in unit test. We used to add Rules 2 times. 
First time on line 47 and second time on the next line using `builder.WithRules([]v1.PolicyRule{rule})` . So as a result we had 2 identical rules